sdk/core/core-amqp/CHANGELOG.md

```diff
@@ -1,6 +1,6 @@
 # Release History
 
-## 1.1.7 (Unreleased)
+## 1.1.7 (2020-10-28)
```

Comment:
https://github.com/Azure/azure-sdk-for-js/pull/11973 introduced the `MessageErrorCodes` which is still under discussion. So, instead of making this change in the master branch, you will need a hotfix branch from the commit before #11973 was merged

Reply:
hotfix branch at cb67b24d57169e04008211c48ba94108ebdb3a81